src/app/[locale]/(pages)/conferma-email/link-scaduto/page.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import { Grid } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { commonBackgroundLightFullHeight } from '../../../_utils/styles';
+import { EmailValidationContainer } from '../../../_component/emailValidationContainer/emailValidationContainer';
+import HourglassIcon from '@/app/[locale]/_icons/hourglass';
+import { backToIo } from '@/app/[locale]/_utils/common';
+
+const EmailConfirmationLinkExpired = (): React.ReactElement => {
+  const t = useTranslations('ioesco');
+  return (
+    <Grid
+      sx={commonBackgroundLightFullHeight}
+      alignItems="center"
+      justifyContent="center"
+      container
+    >
+      <Grid item xs={12} justifySelf={'center'} marginTop={5}>
+        <EmailValidationContainer
+          icon={<HourglassIcon />}
+          title={t('emailvalidation.expiredlinkerrortitle')}
+          summary={t('emailvalidation.expiredlinkerrorsubtitle')}
+          button={{
+            variant: 'contained',
+            text: t('emailvalidation.backtoio'),
+            onClick: () => backToIo(1000),
```

Comment:
What about moving values li `1000` in some shared file? If we use it in many places we could update it at once if it changes. What do you think?